components/segment-bridge/base/serviceaccount.yaml

```diff
@@ -0,0 +1,38 @@
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: segment-bridge-read-access
```

Comment:
missing namespace
you can set one for all the resources on the kustomization file.

Reply:
fixed
will set in the kustomization file once the test will pass... for now I prefer it separated in case changes will be needed (e.g my former comment regarding creating the namespace in dev env)